tests/foreman/cli/test_activationkey.py

```diff
@@ -1202,7 +1197,7 @@ def test_positive_add_subscription_by_id(self):
             u'id': ackey_id,
             u'subscription-id': subs_id[0]['id'],
         })
-        self.assertIn('Subscription added to activation key', result)
+        self.assertIn('Subscription added to activation key.', result)
```

Comment:
if 'Subscription added to activation key' is not in result do not think that 'Subscription added to activation key.' will be, some thing is strange here.

Reply:
No mystery, this part probably didn't fail, it is more about fixing the parts with assertEqual, I just updated all occurencies in the file :)